src/main/java/hudson/plugins/emailext/plugins/recipients/CulpritsRecipientProvider.java

```diff
@@ -51,10 +54,18 @@ public void send(final String format, final Object... args) {
         }
         final Debug debug = new Debug();
         Run<?,?> run = context.getRun();
-        if (run instanceof AbstractBuild) {
-            Set<User> users = ((AbstractBuild<?,?>)run).getCulprits();
-            RecipientProviderUtilities.addUsers(users, context, env, to, cc, bcc, debug);
-        } else {
+        // TODO: core 2.60+, workflow-job 2.12+: Switch to checking if run is RunWithSCM and make catch an else block
+        try {
+            Method getCulprits = run.getClass().getMethod("getCulprits");
+            if (Set.class.isAssignableFrom(getCulprits.getReturnType())) {
+                @SuppressWarnings("unchecked")
+                Set<User> users = (Set<User>) getCulprits.invoke(run);
+                if (Iterables.all(users, Predicates.instanceOf(User.class))) {
+                    RecipientProviderUtilities.addUsers(users, context, env, to, cc, bcc, debug);
+                }
+            }
+        } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
+            debug.send("Exception getting culprits for %s: %s", run, e);
```

Comment:
🐜 this is a valid behavior for 2.60-, no? Maybe it needs more graceful message, but up to you